src/planwise/model/starting_project.cljc

```diff
@@ -0,0 +1,36 @@
+(ns planwise.model.starting-project
+  (:require [clojure.spec.alpha :as s]
+            [clojure.string :refer [blank?]]))
+
+;; ----------------------------------------------------------------------
+;; Project starting validation
+
+;; Goal
+(s/def ::id number?)
+(s/def ::name string?)
+(s/def ::region-id number?)
+
+;; Demographics
+(s/def ::population-source-id number?)
+(s/def ::target number?)
+(s/def ::unit-name (comp not blank?))
+(s/def ::demographics (s/keys :req-un [::unit-name ::target]))
+
+;; Sites
+(s/def ::dataset-id number?)
+(s/def ::capacity number?)
+(s/def ::sites (s/keys :req-un [::capacity]))
+
+;; Coverage
+(s/def ::filter-options map?)
```

Comment:
Even though we cannot be precise with the spec here since we don't have the dataset's coverage algorithm, we may still improve this using `s/or`.